packages/core/src/pages/login.tsx

```diff
@@ -12,7 +13,9 @@ function Page() {
     <>
       <NextSeo noindex nofollow />
 
-      <div>loading...</div>
+      <UIEmptyState title="Loading">
+        <UILoader />
+      </UIEmptyState>
```

Comment:
Fan, do you think we could change this bg to white? Seems a little bit odd with the footer (same thing with the 404 page)

<img width="1586" alt="image" src="https://user-images.githubusercontent.com/11613011/232591268-d2b0b7e2-f468-4075-aa03-957448f0c0e7.png">

Reply:
agreed! Lemme change it!